app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(string: str) -> None:
+    words = string.split()
+    if len(words) != 3:
+        raise TypeError
+    elif words[0] != "mv":
+        raise ValueError
+
+    old_file_name = words[1]
+
+    if words[-1][-1] == "\\" or words[-1][-1] == "/":
+        destination_path = os.path.join(*words[-1])
```

Comment:
The use of `os.path.join(*words[-1])` is incorrect. `os.path.join` should be used with separate arguments, not a single string. Consider splitting the path correctly before joining.